docs/setup.md

```diff
@@ -78,6 +78,7 @@ You have to configure the application to interact with your teams.
    - Windows: Press `Alt` key to bring up the menu at the top of the window, then click `File -> Settings`.
    - OS X: Click `Mattermost` from the menu at the top of the screen, then click `Preferences...`.
    - Linux: Click `File -> Settings` on the menu.
+   - If tray menu : Right-click on tray icon and click `Settings`
```

Comment:
As tray menu is an option and might not be present, i wanted to reflect it and not mislead people on an hidden menu.
Is is too much ?